Assets/SteamVR_Unity_Toolkit/Scripts/Abstractions/VRTK_WorldPointer.cs

```diff
@@ -182,7 +182,9 @@ protected virtual void PointerIn()
             OnDestinationMarkerEnter(SetDestinationMarkerEvent(pointerContactDistance, pointerContactTarget, destinationPosition, controllerIndex));
 
             interactableObject = pointerContactTarget.GetComponent<VRTK_InteractableObject>();
-            if (interactableObject && interactableObject.pointerActivatesUseAction && interactableObject.holdButtonToUse)
+            bool cannot_use_because_not_grabbed = (interactableObject.useOnlyIfGrabbed && !interactableObject.IsGrabbed());
```

Comment:
pascal case your variables (not snake case)

so should be `cannotUseBecauseNotGrabbed`

but i'd also reword this to:

`useOnlyIfGrabbed` (then it's the negative of you original question so just keep that in mind :))